2022-11-17 14:57:02 - INFO - Training/evaluation parameters: Namespace(adam_eps=1e-06, adam_lr=5e-05, base_url='../data/tnews_public/', bert_cache='../config/nghuyong/ernie-1.0-base-zh', bert_dim=768, bert_dir='nghuyong/ernie-1.0-base-zh', bert_max_length=512, bert_padding='max_length', best_score=0.5, ckpt_file='save/BD/v13/model_.bin', data_path='../data/Toursim3.csv', device='cuda', dropout=0.1, log_file='log/BD/v13.log', lstm_batch_first=True, lstm_bidirectional=True, lstm_dropout=0.0, lstm_hidden_size=384, lstm_input_size=768, lstm_num_layers=2, lstm_reflector_size=256, max_epochs=10, max_steps=50000, n_gpu=1, num_class=3, num_workers=6, prefetch=12, print_steps=10, savedmodel_path='save/BD/v13', seed=2021, smoothing=0.0, stopWords='../stopword/hit_stopwords.txt', train_batch_size=24, val_batch_size=24, val_ratio=0.2, warmup_steps=1000, weight_decay=0.001, word2Vec='../word2vec/sgns.weibo.bigram-char.bz2', word2Vec_dim=300)
Some weights of the model checkpoint at nghuyong/ernie-1.0-base-zh were not used when initializing ErnieModel: ['cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.bias']
- This IS expected if you are initializing ErnieModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing ErnieModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
2022-11-17 14:59:44 - INFO - Epoch 0 step 232: loss 0.445, {'accuracy': 0.8113, 'f1': 0.8028, 'precision': 0.8016, 'recall': 0.8113}
2022-11-17 15:02:19 - INFO - Epoch 1 step 464: loss 0.404, {'accuracy': 0.8257, 'f1': 0.8165, 'precision': 0.8174, 'recall': 0.8257}
2022-11-17 15:04:53 - INFO - Epoch 2 step 696: loss 0.549, {'accuracy': 0.7927, 'f1': 0.7955, 'precision': 0.8085, 'recall': 0.7927}
2022-11-17 15:07:26 - INFO - Epoch 3 step 928: loss 0.555, {'accuracy': 0.8135, 'f1': 0.8088, 'precision': 0.8067, 'recall': 0.8135}
2022-11-17 15:10:00 - INFO - Epoch 4 step 1160: loss 0.652, {'accuracy': 0.8056, 'f1': 0.7982, 'precision': 0.7963, 'recall': 0.8056}
2022-11-17 15:12:33 - INFO - Epoch 5 step 1392: loss 0.785, {'accuracy': 0.802, 'f1': 0.8, 'precision': 0.7985, 'recall': 0.802}
Traceback (most recent call last):
  File "main.py", line 99, in <module>
    main()
  File "main.py", line 96, in main
    train_and_validate(args)
  File "main.py", line 54, in train_and_validate
    loss, accuracy, _, _, _ = model(batch)
  File "/home/lenovo/anaconda3/envs/tfNLP/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/lenovo/anaconda3/envs/tfNLP/lib/python3.8/site-packages/torch/nn/parallel/data_parallel.py", line 166, in forward
    return self.module(*inputs[0], **kwargs[0])
  File "/home/lenovo/anaconda3/envs/tfNLP/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/lenovo/wcProject/Bert_ABiLSTM_Att/Code_10.19/model.py", line 164, in forward
    output, (final_hidden_state, _) = self.bilstm(last_hidden_state)
  File "/home/lenovo/anaconda3/envs/tfNLP/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/lenovo/anaconda3/envs/tfNLP/lib/python3.8/site-packages/torch/nn/modules/rnn.py", line 769, in forward
    result = _VF.lstm(input, hx, self._flat_weights, self.bias, self.num_layers,
KeyboardInterrupt
